src/CoState.cs

```diff
@@ -107,4 +107,16 @@ public override void RequestExit()
 			}
 		}
 	}
+
+	public class CoState : CoState<string>
+	{
+		public CoState(
+			Action<CoState<string>> onEnter = null,
+			Func<CoState<string>, IEnumerator> onLogic = null,
```

Comment:
If you just want to shorten the code, you can use the alias using. However, I don't think this is a good solution since it is required for each class that inherits from CoState.

```cs
using StrCoState = FSM.CoState<string, string>;

    // snip

    public class CoState : StrCoState
    {
        public CoState(
            Action<StrCoState> onEnter = null,
            Func<StrCoState, IEnumerator> onLogic = null,
```